source/views/streaming/streams/list.js

```diff
@@ -45,7 +46,7 @@ export class StreamListView extends React.PureComponent {
     error: null,
     loaded: false,
```

Comment:
What is "loaded"? It doesn't seem to be used anywhere.

Reply:
I'll nix it.

Reply:
Nevermind. It's used -- it is for showing the `LoadingView` in render.